### DISCUSSION TITLE
Como adicionar discussao

### DISCUSSION THREAD

**Discussion by CodesdaLu** [community/community#43249] [Discussions] [answered]
### Select Topic Area

Question

### Body

Hey,
how i add discussion in my repository?

**Comment by SilasRodrigues19** [comment, marked as answer]
Oi @CodesdaLu 

É só você acessar as configurações do repositório, vai ter uma aba `"Features/Recursos"` e um pouco abaixo tem `"Discussions/Discussão" ` com uma checkbox do lado para marcar e ativar.

Você pode por exemplo acessar esse link:
https://github.com/CodesdaLu/Programming-Books/settings

Dar um CTRL F e pesquisar por `"Discussão" ` ou  `"Discussions"` se tiver em inglês, deve aparecer algo assim:

![image](https://user-images.githubusercontent.com/47532703/210445898-8fd243bf-28e6-43b1-9168-852ea1063349.png)

**Reply by SilasRodrigues19** [reply]
Depois é só clicar no botão verde ali e configurar do jeito que você quer, ou usar o modelo pronto que tem

**Comment by CodesdaLu** [comment]
Obrigada Silas
